blazor/pager/behaviour-settings.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: Behaviour Settings in Blazor Pager Component | Syncfusion
+description: Checkout and learn here all about basic functionalites of render the Syncfusion Blazor Pager component and much more.
```

Comment:
description: Checkout here and learn about all the basic functionalities of rendering the Syncfusion Blazor Pager component and much more.